var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class Timemory(CMakePackage):
+    """Timing + Memory + Hardware Counter Utilities for C/C++/CUDA/Python"""
+
+    homepage = 'https://timemory.readthedocs.io/en/latest/'
+    git = 'https://github.com/NERSC/timemory.git'
+
+    version('master', branch='master', submodules=True)
+
+    variant('python', default=True, description='Enable Python support')
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('papi', default=True, description='Enable PAPI support')
+    variant('cuda', default=True, description='Enable CUDA support')
+    variant('cupti', default=True, description='Enable CUPTI support')
+    variant('caliper', default=True, description='Enable Caliper support')
+    variant('gperftools', default=True, description='Enable gperftools support')
+
+    extends('python', when='+python')
+    depends_on('python@3:', when='+python', type=('build', 'run'))
+    depends_on('py-numpy', when='+python', type=('run'))
+    depends_on('py-pillow', when='+python', type=('run'))
+    depends_on('py-matplotlib', when='+python', type=('run'))
```

Comment:
Where do these dependencies come from? They aren't in the `setup.py`. If you're looking in `requirements.txt`, then you forgot `py-scikit-build`.

Reply:
I don't need `py-scikit-build` because I am running CMake directly. That `setup.py` only exists for PyPi and Anaconda builds -- I resolve the path for the python installation internally in CMake is scikit-build is not used.